interface/ceed-jit-tools.c

```diff
@@ -205,7 +205,7 @@ int CeedLoadSourceToInitializedBuffer(Ceed ceed, const char *source_file_path, c
 **/
 int CeedLoadSourceToBuffer(Ceed ceed, const char *source_file_path, char **buffer) {
   // Initialize buffer
-  CeedCall(CeedCalloc(1, buffer));
+  CeedCall(CeedCalloc(1, &buffer));
```

Comment:
I'm a little confused by adding `&` here, and I think CI doesn't like it

Reply:
Oh, oops. I had first adjusted things to put the `const` here and use a local variable, then backpedaled and moved it up one frame, but forgot to revert this line.